pylot/perception/flags.py

```diff
@@ -39,13 +39,16 @@
 
 # Deep sort tracking flags.
 flags.DEFINE_string(
-    'deep_sort_tracker_person_weights_path',
-    'dependencies/models/tracking/deep-sort-carla/ped_feature_extractor',
-    'Path to weights for person feature extractor model')
+    'deep_sort_tracker_weights_path',
+    'dependencies/models/tracking/deep-sort-carla/feature_extractor',
+    'Path to weights for feature extractor model')
 
 # Object tracking flags.
 flags.DEFINE_bool('visualize_tracker_output', False,
                   'True to enable visualization of tracker output')
+flags.DEFINE_float(
```

Comment:
Can you please rebase and move these flag updates to https://github.com/erdos-project/pylot/blob/master/pylot/perception/flags.py#L41 ?